src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-sidebar/dot-edit-layout-sidebar.component.ts

```diff
@@ -32,7 +33,10 @@ export class DotEditLayoutSidebarComponent implements OnInit, ControlValueAccess
     ) {}
 
     ngOnInit() {
-        this.dotMessageService.getMessages(['editpage.layout.designer.sidebar']).subscribe();
+        this.dotMessageService
+            .getMessages(['editpage.layout.designer.sidebar'])
+            .pipe(take(1))
```

Comment:
I think this take(1) is not need because the messageService do a complete

https://github.com/dotCMS/core-web/blob/master/src/app/api/services/dot-messages-service.ts#L78

Reply:
if what rori says is true, then here https://github.com/dotCMS/core-web/pull/976/files#diff-b34ca7e680752f5ef53f1b80ca940b6d is the same thing!